### DISCUSSION TITLE
Custom Registry Support for Dart

### DISCUSSION THREAD

**Discussion by usison** [renovatebot/renovate#25984] [Suggest an Idea]
### Tell us more.

In the docs for the Renovatbot it says that custom registries are not supported for Dart. Are there any attempts to support custom registries for Dart or  are there any workarounds to make this work. We have some Dart packages which are hosted in our private Artifactory and it would be great if Renovate would also work on this.

**Comment by rarkins** [comment]
Here is the relevant code which is used to generate the documentation: https://github.com/renovatebot/renovate/blob/3d47d2a615470a00ab6c5d606fabda42e160a1c3/lib/modules/datasource/dart/index.ts#L13-L15

I don't think there's any harm in us switching that from false to true, however it's possible that Artifactory's API is not identical to the public dartlang server and therefore it may not work "out of the box" and could need some adjustments by you or someone else who can test it against Artifactory.

**Reply by usison** [reply]
Thanks for the quick response that sounds great. From what i understand is that it might happen that you will change the boolean customRegistrySupport to true and we need to test this change thoroughly on our end and see if this works  "out of the box ™"
